packages/mds-compliance-engine/engine/helpers.ts

```diff
@@ -0,0 +1,181 @@
+import {
```

Comment:
It looks like some of the new files are missing license headers. I recommend installing the [licenser](https://github.com/ymotongpoo/vsc-licenser) extension that is already configured to insert the header automatically in new files in this repo. Also, it can be used to manually add the properly formatted header to existing files.